### ISSUE TITLE
Reader/Xlsx Chart Problems, Probably Due to Namespacing Changes

### ISSUE THREAD

**Issue by oleibman** [PHPOffice/PhpSpreadsheet#2203]
This is:

```
- [x] a bug report
- [ ] a feature request
- [x] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?
Samples spreadsheets 32* look wrong when created with master. They appear okay with 18.0. Strongly suspect the problem is due to the namespacing changes in Reader/Xlsx. I am researching.

### What is the current behavior?
Charts are wrong.

### What are the steps to reproduce?
Run Samples/32_Chart_read_write.php

### Which versions of PhpSpreadsheet and PHP are affected?
18.0 unaffected, master affected.

**Comment by oleibman** [comment]
The chart is constructed with the correct set of parameters in Reader/Xlsx (using 32readwriteAreaChart1.xlsx). When Writer recalculates the values, they wind up being reset to null string. No idea why. When I stop the recalculation, it still doesn't work. So multiple problems are happening. I'll start on it again tomorrow, probably with a different spreadsheet that doesn't use random numbers, in order to simplify things.

**Comment by oleibman** [comment]
Interesting - new code works correctly with non-random data. But it seems to fail if a formula is used in any cell.

**Comment by oleibman** [comment]
It seems that reading the chart is causing all the formulas on the sheet to go bad, whether or not involved in the chart.

**Comment by oleibman** [comment]
It appears the code is not handling shared formulas properly. I need to study.

**Comment by oleibman** [comment]
Fixed by #2204. Charts turn out to have nothing to do with the problem; the sample chart spreadsheets just happened to expose the real problem, which was that shared formulas were not being handled correctly.
